apps/betterangels-backend/hmis/errors.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
```

Comment:
I don't think this is necessary anymore.  Since 3.11 i believe annotations are turned on be default.